Monika-After-Story/MonikaModDev
# Proper return keys

Changed our return keys from `str`s to `dict`s to use the `kwargs` format. It's just cleaner and makes managing things like idle displayable easier (and faster w/o heavy regex). I kept support for the old format, let's give a release or two so people can update their submods/etc.

**I'd like to get this merged sooner than later (added `high priority`), because each new topic would require updating this PR (and it's easy to miss something).**

### Changes:
- New format for return keys.

### Testing:
- Verify each return key works as expected.
- Verify the return keys work in pairs.
- Scroll through the changes tab and make sure I haven't made a typo.
- Ideally look through the scripts and verify we updated all instanced where we use return keys.
